app/src/main/java/org/p2p/wallet/auth/ui/phone/countrypicker/CountryCodePickerFragment.kt

```diff
@@ -77,13 +72,13 @@ class CountryCodePickerFragment :
                 presenter.search(emptyString())
             }
         })
-        searchView.openSearch()
+        setTitle(R.string.striga_country)
```

Comment:
if it's reused in Striga and Onboarding let's move this fragment and update title depending on place